### DISCUSSION TITLE
Как запускать под windows?

### DISCUSSION THREAD

**Discussion by ivvaan** [hufrea/byedpi#79] [Q&A]
В релизе 4 bat файла:
service_install.bat
service_restart.bat
service_delete.bat
byedpi.bat

Что нужно сделать, чтобы работало?
1. Запустить byedpi.bat? А для чего тогда service_install.bat?
2. Запустить service_install.bat? А для чего тогда byedpi.bat?
3. Запустить сначала service_install.bat, а затем  byedpi.bat?

Все обсуждают аргументы командной строки, а как вообще правильно запустить приложение не понятно.

**Comment by lufog** [comment]
- byedpi.bat - запустить как программу. В панели задач будет консольное окно, при закрытии которого программа остановится. После каждой перезагрузки программу нужно запускать по новой.
- service_install.bat - добавить и запустить как службу Windows. Никаких окон в панели задач, автоматически запускается при загрузке Windows.
- service_restart.bat - перезапустить службу Windows. (Это можно сделать и через UI в Службах)
- service_delete.bat - остановить и удалить службу Windows. Например, перед удалением программы (если до этого служба Windows была установлена).

[Служба Windows](https://ru.wikipedia.org/wiki/%D0%A1%D0%BB%D1%83%D0%B6%D0%B1%D0%B0_Windows)

**Comment by jura1243** [comment]
можно запустить убунту в multipass и туда обращаться из firefox. другие броузеры у меня не заработали.
